tests/hikari/interactions/test_component_interactions.py

```diff
@@ -136,28 +136,3 @@ def test_get_guild_when_cacheless(self, mock_component_interaction, mock_app):
         assert mock_component_interaction.get_guild() is None
 
         mock_app.cache.get_guild.assert_not_called()
-
-    @pytest.mark.asyncio()
-    async def test_fetch_parent_message(self):
-        stub_interaction = hikari_test_helpers.mock_class_namespace(
-            component_interactions.ComponentInteraction, fetch_message=mock.AsyncMock(), init_=False
-        )()
-        stub_interaction.message = mock.Mock(id=3421)
-
-        assert await stub_interaction.fetch_parent_message() is stub_interaction.fetch_message.return_value
-
-        stub_interaction.fetch_message.assert_awaited_once_with(3421)
-
-    def test_get_parent_message(self, mock_component_interaction, mock_app):
-        mock_component_interaction.message = mock.Mock(id=321655)
-
-        assert mock_component_interaction.get_parent_message() is mock_app.cache.get_message.return_value
-
-        mock_app.cache.get_message.assert_called_once_with(321655)
-
-    def test_get_parent_message_when_cacheless(self, mock_component_interaction, mock_app):
-        mock_component_interaction.app = mock.Mock(traits.RESTAware)
-
-        assert mock_component_interaction.get_parent_message() is None
-
-        mock_app.cache.get_message.assert_not_called()
```

Comment:
Think we can keep the test until its removed to make sure we dont break the behavior before its removed